src/lib/validation/tests/_support.js

```diff
@@ -0,0 +1,72 @@
+'use strict'
+
```

Comment:
If you have test helper files please add it to test/util and add appropriate name